cloudfoundry/resource_cf_app.go

```diff
@@ -254,9 +260,32 @@ func resourceApp() *schema.Resource {
 							Type:     schema.TypeString,
 							Optional: true,
```

Comment:
Looking at this again, I think it would be best to flag this `validation_script` field as removed as well.  It's removal shouldn't be a problem as it wasn't implemented anyways. Also, it is not currently documented.

Reply:
+1